gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -454,7 +458,7 @@ struct SkipList {
             return xor_sums;
     }
 
-    // Get the sum of the entire sequence
+    // Get the sum of the entire sequence //Related to Connectivity
```

Comment:
Do you mean to nest a comment within a comment here? That's a bit confusing? Should it be on a separate line? Do you mean to use some other punctuation?